src/main/java/utils/Constants.java

```diff
@@ -0,0 +1,24 @@
+package utils;
+
+import java.util.Random;
+
+public class Constants {
+
+    private static Random random = new Random();
+    public static final String LOGIN_PAGE_URL = "https://zenistock.zenitech.local/login";
+    public static final String ADMIN_USERNAME = "admin";
+    public static final String ADMIN_PASSWORD = "qazqwer1@d.";
+    public static final String DEVICE_NAME = "Laptop DELL";
+    public static final String DEVICE_SERIAL_NO = random.nextInt(100000) + "";
+    public static final String DEVICE_INVENTORY_NO = random.nextInt(100000) + "";
+    public static final String DEVICE_INVOICE_NO = random.nextInt(1000000) + "";
```

Comment:
do we need to concatenate the random number with an empty string?

Reply:
the "sendKeys" method doesn't acceot arguments of type long, and that is why I had to convert them to strings. This seemed the simplest option to do so